_shared/project/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+concurrency:
+  group: deploy
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    uses: ./.github/workflows/ci.yml
+  Docker_Hub:
+    name: Docker Hub
+    needs: CI
+    uses: hypothesis/workflows/.github/workflows/dockerhub.yml@main
+    with:
+      Application: {% raw %}${{ github.event.repository.name }}{% endraw +%}
+    secrets: inherit
+  {% if include_exists(".github/workflows/environments.json") %}
+  {% for job_id, environment in include_json(".github/workflows/environments.json").items() %}
+  {{ job_id }}:
+    {% if "github_environment_name" in environment %}
+    name: {{ environment.github_environment_name }}
+    {% endif %}
+    needs: [{{ (["Docker_Hub"] + environment.get("needs", []))|join(", ") }}]
```

Comment:
```suggestion
    needs: [{{ (["Docker_Hub"] + environment.get("needs", [])) | join(", ") }}]
```

Reply:
I'm gonna follow the style used in the Jinja2 docs and continue to not put spaces around `|`'s